src/components/Table/ImageTable.js

```diff
@@ -171,8 +171,8 @@ const ImageTable = (props) => {
   const intl = useIntl();
   // start sorting by created date
   const [activeSortIndex, setActiveSortIndex] = useState(null);
```

Comment:
You should also set the activeSortIndex to something related to the "Created" column. I think you could combinw both `fieldId` and `culumnIndex` in the sort handlers.

Reply:
I've updated the `activeSortIndex`